components/clp-mcp-server/tests/server/test-utils.py

```
@@ -0,0 +1,130 @@
+"""Unit tests for CLP MCP server's utility functions."""
```

Ruff (0.14.1)

Invalid module name: 'test-utils'

(N999)